tools/ai2_common.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+# 0-*- Mode: Python; tab-width: 4; indent-tabs-mode: nil; coding: utf-8; -*-
+
+import sys
+import os.path
+import spacy
+import itertools
+import glob
+import re
+from sys import path as sys_path
+
+# this seems to be necessary for annotations to find its config
+sys_path.append(os.path.join(os.path.dirname(__file__), '..'))
+try:
+    import annotation
```

Comment:
Never mind, found it https://github.com/allenai/brat/blob/master/server/src/annotation.py